packages/http/test/http-decorators.test.ts

```diff
@@ -274,8 +287,8 @@ describe("http: decorators", () => {
       `);
       expectDiagnostics(diagnostics, [
         {
-          code: "@typespec/http/shared-boolean",
-          message: `shared parameter must be a boolean.`,
+          code: "invalid-argument",
+          message: `Argument '(anonymous model)' is not assignable to parameter of type '(anonymous model)'`,
```

Comment:
This seems pretty gross, but I guess if it's going away anyhow it doesn't matter too much 😆

Reply:
Yep, this error needs to be improved, filed an issue about it.